polybar-scripts/pulseaudio-microphone/pulseaudio-microphone.sh

```diff
@@ -25,16 +25,32 @@ toggle() {
   DEFAULT_SOURCE=$(pacmd list-sources | awk '/\*/,EOF {print $3; exit}')
 
   if [ "$MUTED" = "yes" ]; then
-      pacmd set-source-mute "$DEFAULT_SOURCE" 0
+      pactl set-source-mute "$DEFAULT_SOURCE" 0
   else
-      pacmd set-source-mute "$DEFAULT_SOURCE" 1
+      pactl set-source-mute "$DEFAULT_SOURCE" 1
```

Comment:
Why du you use pactl instead of pacmd? 

As pacmd is already part of the pulseaudio package, pacmd would be a new dependency. So we could use it, but to have fewer requirements is also a nice goal.

Reply:
I believe the original reason for using `pactl` instead of `pacmd` is that it does not allow you to increase the volume by a certain percent - `pactl set-source-volume "$DEFAULT_SOURCE" +5%`

Reply:
Thats what I had in my head too. Alright then leets keep it like this.